iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -235,19 +237,14 @@ def createViewer(self, scriptUrl="js/dist/ixbrlviewer.js"):
 
             if f.context.isForeverPeriod:
                 aspects["p"] = "f"
-            elif f.context.isInstantPeriod and f.context.instantDatetime is not None:
+            elif f.context.isInstantPeriod:
                 aspects["p"] = self.dateFormat(f.context.instantDatetime.isoformat())
-            elif f.context.isStartEndPeriod and f.context.startDatetime is not None and f.context.endDatetime is not None:
```

Comment:
Taking away both of these checks here seems slightly dangerous knowing that garbage data will run through here.

Reply:
Yes, I agree.  My GitHub view was stale, so I ended up making a similar comment [here](https://github.com/Workiva/ixbrl-viewer/pull/74#discussion_r365251531).

Reply:
old copy, added these in